traffic_ops/app/db/migrations/2021011500000000_update_interfaces_multiple_routers.sql

```diff
@@ -0,0 +1,53 @@
+/*
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+ALTER TABLE interface ADD COLUMN router_host_name text DEFAULT '';
+ALTER TABLE interface ADD COLUMN router_port text DEFAULT '';
+
+UPDATE interface
+SET router_host_name = CASE
+WHEN server.router_host_name IS NULL THEN ''
+ELSE interface.router_host_name || server.router_host_name
+END
```

Comment:
This looks like it'll work fine, but for what it's worth it looks like you wanted a `COALESCE` here. Since you just created the router_port column of an interface, they can't already have any text, so this should be equivalent to
```sql
UPDATE interface
SET router_host_name = CASE
WHEN server.router_host_name IS NULL THEN ''
ELSE server.router_host_name
END
FROM server
WHERE server = server.id;
```

which, in turn, is equivalent to

```sql
UPDATE interface
SET router_host_name = COALESCE(server.router_host_name, '')
FROM server
WHERE server = server.id;
```

... but also, can't `interface.router_host_name` itself be NULL? Then maybe you don't need to check at all. If the intention was to make it NOT NULL then I think you missed a constraint.